### PR TITLE
Add 2009scape Launcher

### PR DESCRIPTION
**Pull request by dginovker**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by dginovker** [comment]
bot, build org._2009scape.Launcher

**Comment by dginovker** [comment]
bot, build org._2009scape.Launcher

**Comment by dginovker** [comment]
bot, build org._2009scape.Launcher

**Comment by dginovker** [comment]
App works as expected

File: org._2009scape.Launcher.json [hunk 1]
```diff
@@ -0,0 +1,66 @@
+{
+    "id": "org._2009scape.Launcher",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
```
Comment (hfiguiere) [inline org._2009scape.Launcher.json:4]:
```suggestion
    "runtime-version": "22.08",
```

File: org._2009scape.Launcher.json [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.openjdk11"
+    ],
+    "finish-args": [
+        "--share=network",
+        "--socket=x11",
+        "--share=ipc",
+        "--device=dri",
+        "--socket=pulseaudio",
+        "--filesystem=home/.local/share/2009scape",
```
Comment (hfiguiere) [inline org._2009scape.Launcher.json:17]:
```suggestion
        "--filesystem=xdg-data/2009scape",
```

Assuming you really need to access this.

File: org._2009scape.Launcher.json [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+                {
+                    "type": "script",
+                    "dest-filename": "2009scape-launcher",
+                    "commands": [
+                          "exec 2009scape-launcher \"$@\""
+                    ]
+                },
+                {
+                    "type": "file",
+                    "url": "https://gitlab.com/2009scape/Saradomin-Launcher/-/jobs/artifacts/master/raw/Saradomin/2009scape-launcher-sc?job=pack_linux_sc",
```
Comment (hfiguiere) [inline org._2009scape.Launcher.json:45]: this and the other should NOT point to `master` but rather to a **specific** revision.

also this seems to be DOTnet based, but I don't see a runtime installed.

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: org._2009scape.Launcher.json (hunk 3 above)
Comment (dginovker) [reply org._2009scape.Launcher.json:45]: It's Dotnet based, but it has all the dependencies included (so it works). There's a non-dependency included version, I'll see if I can get that one working with the Flatpak runtime :+1:

Comment (dginovker) [reply org._2009scape.Launcher.json]: I'm probably doing something wrong but I wasn't able to get the non-dependency version working with either the Mono or Dotnet extension :/. In any case, this Flatpak works on a brand new Fedora Silverblue install so it seems to be fine

File: org._2009scape.Launcher.json (hunk 2 above)
Comment (dginovker) [reply org._2009scape.Launcher.json]: Thanks :+1:

**Comment by dginovker** [comment]
bot, build org._2009scape.Launcher

**Comment by dginovker** [comment]
Looks good - Ready for another review :+1:

File: org._2009scape.Launcher.json (hunk 3 above)
Comment (hfiguiere) [reply org._2009scape.Launcher.json]: so you are packaging an x86_64 binary on aarch64.

Comment (dginovker) [reply org._2009scape.Launcher.json]: I did some quick research, and it seems like you can't natively run x86_64 binaries on aarch64. I'll exclude aarch64

**Comment by dginovker** [comment]
Thanks for all your help @hfiguiere :heart: :taco:

**Pull request closed without merging**